### ISSUE TITLE
Hibernation support for windows 8 / server 2012

### ISSUE THREAD

**Issue by iMHLv2** [volatilityfoundation/volatility#25]

**Comment by nirizr** [comment]
I'm curious to know what's the status of this task. I'm a professional reverse engineer and might be willing to lend a hand with figuring out the new/modified hiberfile.sys format.

I've previously contributed to Volatility with a somewhat similar task ([VMWare snapshots](https://github.com/volatilityfoundation/volatility/wiki/VMware-Snapshot-File)) if you consider this something valuable.

**Comment by jared703** [comment]
Agreed, this would be awesome Nir. Thank you for your contributions for VM
related snapshots. They are extremely valuable to analysts like myself.
